src/components/password-reset-provider.tsx

```diff
@@ -0,0 +1,47 @@
+import { Password } from "./forms/password";
+import { Email } from "./forms/email";
+
+export function PasswordResetProvider({
+  if: condition,
+  setPassword: setPassword,
+  setConfirmPassword: setConfirmPassword,
+  password: password,
+  confirmPassword: confirmPassword,
+  email: email,
+  setEmail: setEmail,
+  isValidEmail: isValidEmail,
+  setIsValidEmail: setIsValidEmail,
+}: {
+  if: boolean,
+  setPassword: (password: string) => void,
+  setConfirmPassword: (confirmPassword: string) => void,
+  password: string,
+  confirmPassword: string,
+  email: string,
+  setEmail: (email: string) => void,
+  isValidEmail: boolean,
+  setIsValidEmail: (isValidEmail: boolean) => void,
+}) {
+  if (condition)
+    return (
+      <>
+        <div className="flex justify-end items-center relative my-5">
```

Comment:
This will cause a centering/size issue
![Capture d’écran du 2023-11-05 19-56-47](https://github.com/Code-Society-Lab/cursif-web/assets/49426163/06df5771-ad1c-4482-b546-00dbfbc7ba36)